transmart-data/ddl/postgres/i2b2demodata/qt_analysis_plugin.sql

```diff
@@ -30,3 +30,8 @@ ALTER TABLE ONLY qt_analysis_plugin
 --
 CREATE INDEX qt_apnamevergrp_idx ON qt_analysis_plugin USING btree (plugin_name, version_cd, group_id);
 
+--
+-- add documentation
+--
+COMMENT ON TABLE i2b2demodata.observation_fact IS 'Not used by TranSMART. The plug-in’s metadata for an individual project is stored in this
```

Comment:
Comment is on wrong table. That one is actually used by transmart...
Maybe remove this comment entirely?